File: fontend/src/types/enums.ts
// 事件状态
export const EventStatus = {
  DRAFT: 'draft',
  WAITING_B: 'waiting_B',
  JUDGED: 'judged',
  REVIEWED: 'reviewed',
  CLOSED: 'closed',
} as const;

export type EventStatus = typeof EventStatus[keyof typeof EventStatus];

// 用户状态
export const UserStatus = {
  ACTIVE: 'active',
  LOCKED: 'locked',
  DISABLED: 'disabled',
} as const;

export type UserStatus = typeof UserStatus[keyof typeof UserStatus];

// 消息角色
export const MessageRole = {
  USER: 'user',
  ASSISTANT: 'assistant',
  SYSTEM: 'system',
} as const;

export type MessageRole = typeof MessageRole[keyof typeof MessageRole];

File: backend/app/constants/enums.py
from enum import Enum


class UserStatus(str, Enum):
    ACTIVE = "active"
    LOCKED = "locked"
    DISABLED = "disabled"


class RelationshipStatus(str, Enum):
    PENDING = "pending"
    ACTIVE = "active"
    CLOSED = "closed"


class EventStatus(str, Enum):
    DRAFT = "draft"
    WAITING_B = "waiting_b"
    JUDGED = "judged"
    REVIEWED = "reviewed"
    CLOSED = "closed"


class SnapshotSide(str, Enum):
    A = "a"
    B = "b"


class ModerationRiskLevel(str, Enum):
    LOW = "low"
    MEDIUM = "medium"
    HIGH = "high"
